bin/amphtml-update.sh

```diff
@@ -1,12 +1,21 @@
 #!/bin/bash
-set -e
+# Update includes/sanitizers/class-amp-allowed-tags-generated.php based on the AMPHTML validator spec.
+#
+# To update to the latest release of AMPHTML:
+#
+# $ ./amphtml-update.sh
+#
+# Update to a specific commit of AMPHTML
+#
+# $ git clone git@github.com:ampproject/amphtml.git amphtml
+# $ cd amphtml; git checkout ec5fd60; cd -
+# $ ./amphtml-update.sh amphtml/
 
-# Go to the right location.
-cd "$(dirname "$0")"
+set -e
 
-BIN_PATH="$(pwd)"
-PROJECT_PATH=$(dirname $PWD)
-VENDOR_PATH="$PROJECT_PATH/vendor"
+BIN_PATH="$(dirname "$0")"
+PROJECT_PATH=$(dirname $BIN_PATH)
+AMPHTML_LOCATION="$1"
```

Comment:
Can we use `/amphtml` as the default here in case no argument is provided?

Reply:
When no argument is provided, then the plugin just downloads the tarball into a temp directory and then deletes the temp directory when done. We don't want a persistent location in the normal case, as then we'd not be getting the latest if `amphtml` already exists.